tests/test_analyses.py

```diff
@@ -44,11 +44,22 @@ def test_roc(client, test_dataset):
     }
     resp = client.get(url_for('csv.get_roc', csv_id=str(csv_id)), json=data)
     assert resp.status_code == 200
-    keys = {'sensitivities', 'thresholds', 'specificities', 'auc'}
+    keys = {
+        'sensitivities',
+        'thresholds',
+        'specificities',
+        'auc',
+        'lower_bound',
+        'upper_bound',
+        'y'
```

Comment:
`y` is much less descriptive than the other values. Is there a better name we could use?

Reply:
It's provided from the R library, so not really. It's also not used on the frontend, so I don't think it's very significant. It can be removed from the python code that calls the R analysis if you think it's misleading.